### DISCUSSION TITLE
v2ray-core不能启动

### DISCUSSION THREAD

**Discussion by Anthony6075** [v2rayA/v2rayA#1441] [Q&A]
不知怎么，v2raya三四天之前突然不能用了，看日志好像只有一条`[I] [main.go:30] the core was not running the last time v2rayA exited`相关，报告v2ray-core不能运行，同时使用ps命令检查，也没有v2ray-core的进程，而只有v2raya的进程。

v2raya版本2.2.5.6, v2ray-core版本5.16.1，系统ubuntu22.04

全部日志如下:
2024/07/05 15:22:32.242 [A] [main.go:29] V2Ray binary is /usr/bin/v2ray
2024/07/05 15:22:32.242 [A] [main.go:29] V2Ray asset directory is /run/user/0/v2raya
2024/07/05 15:22:32.242 [A] [main.go:29] v2rayA working directory is /
2024/07/05 15:22:32.242 [A] [main.go:29] v2rayA configuration directory is /etc/v2raya
2024/07/05 15:22:32.242 [A] [main.go:29] Golang: go1.22.4
2024/07/05 15:22:32.242 [A] [main.go:29] OS: linux
2024/07/05 15:22:32.242 [A] [main.go:29] Arch: amd64
2024/07/05 15:22:32.242 [A] [main.go:29] Lite: false
2024/07/05 15:22:32.242 [A] [main.go:29] Version: 2.2.5.6
2024/07/05 15:22:32.242 [A] [main.go:29] Starting...
2024/07/05 15:22:32.242 [I] [main.go:30] the core was not running the last time v2rayA exited
2024/07/05 15:22:32.246 [A] [index.go:116] v2rayA is listening at http://127.0.0.1:2017
2024/07/05 15:22:32.246 [A] [index.go:116] v2rayA is listening at http://192.168.1.6:2017
2024/07/05 15:22:32.246 [A] [index.go:116] v2rayA is listening at http://[::1]:2017
2024/07/05 15:22:32.246 [A] [index.go:116] v2rayA is listening at http://[2408:8207:1866:470:a051:c869:3b0e:c0de]:2017
2024/07/05 15:22:32.246 [A] [index.go:116] v2rayA is listening at http://[2408:8207:1866:470:eef4:191e:4376:76fe]:2017
2024/07/05 15:22:32.246 [A] [index.go:116] v2rayA is listening at http://[fe80::a55c:a869:2977:dad2]:2017

**Comment by Zmmfly** [comment]
win11也是这样

**Reply by seagle0128** [reply]
docker也是这样

**Comment by aohanhongzhi** [comment]
怎么解决的啊？

**Reply by aohanhongzhi** [reply]
<img width="1473" alt="image" src="https://github.com/user-attachments/assets/43f9fe4a-491b-4ab8-b101-44d1f7816680" />
